src/Microsoft.AspNetCore.Antiforgery/Internal/DefaultAntiforgeryTokenGenerator.cs

```diff
@@ -66,7 +66,12 @@ public AntiforgeryToken GenerateCookieToken()
             if (identity != null && identity.IsAuthenticated)
```

Comment:
Note that if `Identity` is equal to the first member of `Identities` where `IsAuthenticated` is `true`, existing code was :ok:. (I'm not sure whether "primary" in the `Identity` doc comment means `IsAuthenticated` or something else.)